pages/workspace/tracking-plan/webhook-signing.mdx

```diff
@@ -0,0 +1,65 @@
+---
+layout: 'guide'
+title: 'Webhook signing'
+# abstract: 'Learn about how to secure your webhook endpoint'
+---
+
+import Link from '../../../components/Link';
+
+
+## Check the webhook signatures
+
+### Verify the that only Avo is sending you events!
+
+Avo will sign each webhook call it sends to your endpoint by including a signature in each request `Avo-Signature` header. 
+This allows you to verify that the events were sent by Avo and not by a third party. 
+Before you can verify signatures, you need to retrieve your publishing webhook's secret from your <publishing page>.
+Avo generates a unique secret key for each publishing webhook endpoint. If you use multiple endpoints, you must obtain a secret for each one you want to verify signature on.
+<p>
+  <img
+    src={require('../../../images/signing-secret.png')}
+    width="200px"
+    alt="Signing secret for webhook publishing"
+/>
+</p>
+
+
+### Verifying signatures
+
+The `Avo-signature` header included in each request contains a timestamp and a signature. 
+The timestamp is prefixed by `ts=` and the signature is prefixed by `v1`.
+
+```
+Avo-Signature:
+ts=1651677541,
+v1=30703093db3c532c6f0ab1f1f9eb80b134f5bf6e2ff7064c403aab7582df0667
+```
+
+Avo generates signatures using a hash-based message authentication code (HMAC) with SHA-256.
+
+### Verify signature header
+
+1. Extract the timestamp and signature from the header
+
+Split the header on comma(,) as seperator to get a list of elements, Then split each element using the = to get a key and value dict. 
+You should now have a dict of 2 key value pairs, 
+```{
+    ts:1651677541,
+    v1:30703093db3c532c6f0ab1f1f9eb80b134f5bf6e2ff7064c403aab7582df0667
+}```
```

Comment:
Should this be like this? Don’t know if this matters in the grand scheme of things but it seems to mess up the github preview of it. Sorry, don’t know how to escape the code block out
```suggestion
\```
{
    ts:1651677541,
    v1:30703093db3c532c6f0ab1f1f9eb80b134f5bf6e2ff7064c403aab7582df0667
}
\```
```